code/go/0dns.io/core/config/config.go

```diff
@@ -23,7 +23,7 @@ func SetupConfig() {
 	viper.AddConfigPath("./config")
 	err := viper.ReadInConfig() // Find and read the config file
 	if err != nil {             // Handle errors reading the config file
-		panic(fmt.Errorf("fatal error config file: %s", err))
+		panic(errors.Wrap(err, "fatal error with config file"))
```

Comment:
Please use go standard errors instead of our own error wrapper. for 0dns, standard errors is enough.  https://blog.golang.org/go1.13-errors